src/_js/lib/DynamicLoad.js

```diff
@@ -1,112 +1,77 @@
 import { fullPath } from './Helpers';
 
-let currentPathName = document.location.pathname;
-
-const replaceContent = function(html) {
-  const $page = $(html.replace('<!DOCTYPE html>', ''));
-  const $new = $page.find('[data-dynamic-load]');
-  const $title = $page.filter('title').eq(0);
-  $('head title').text($title.text());
-
-  $new.each(function(i, el) {
-    const $el = $(el);
-    const slug = $el.data('dynamic-load');
-    const content = $el.html();
-    $(`[data-dynamic-load="${slug}"]`).html(content);
-  });
-};
-
-const isSameDomain = function(here, there) {
-  return here.protocol === there.protocol && here.host === there.host;
+const isDifferentDomain = function(here, there) {
+  return here.protocol !== there.protocol && here.host !== there.host;
```

Comment:
`||` instead of `&&`, as this should return `false` when either or both are false.